Show Created/Updated dates on Marketplace entries

sebthom opened the pull request:
- Appends "Updated … | Created …" to the installs row in DiscoveryItem
- Reads existing `INode.getChanged()/getCreated()` values; no additional service calls
- Hides fields gracefully when values are missing

Rationale: helps users make informed decisions between similar plugins by showing maintenance recency (actively maintained vs. stale) and whether something is very new (potentially less stable).

<img width="797" height="868" alt="image" src="https://github.com/user-attachments/assets/974959af-4f75-4644-a784-f025778df438" />

l3-g5 commented on org.eclipse.epp.mpc.ui/src/org/eclipse/epp/internal/mpc/ui/wizards/DiscoveryItem.java:
```diff
@@ -218,6 +223,31 @@ protected void createInstallInfo(Composite parent) {
+			final var dateFormat = DateFormat.getDateInstance(DateFormat.SHORT);
+			final var datesTextBuilder = new StringBuilder();
+			if (changed != null) {
+				if (!datesTextBuilder.isEmpty()) {
```
Since it is a new StringBuilder, this should always be empty. Could be removed.

sebthom committed:
Show Created/Updated dates on Marketplace entries

- Appends "Updated … | Created …" to the installs row in DiscoveryItem
- Reads INode.getChanged()/getCreated(); no additional service calls
- Hides fields gracefully when values are missing

Rationale: helps users make informed decisions between similar plugins by showing maintenance recency (actively maintained vs. stale) and whether something is very new (potentially less stable).

sebthom commented:
@l3-g5 is anything else missing?
